sno/merge_util.py

```diff
@@ -221,10 +223,9 @@ def _deserialise_conflict_part(cls, index_entry):
         result = AncestorOursTheirs.partial(**{version: result_entry})
         return key, result
 
-    def conflicts_as_entries(self):
-        for key, conflict3 in self.conflicts.items():
-            for index_entry in self._serialise_conflict(key, conflict3):
-                yield index_entry
+    def _conflicts_paths(self):
+        """All the paths in all the entries in all the conflicts, as a set."""
+        return set(e.path for c in self.conflicts.values() for e in c if e)
```

Comment:
👎 double comprehensions are really confusing, can you split this into a proper for loop?

```python
from .utils import ungenerator

@ungenerator(set)
def _conflicts_paths(self):
    """All the paths in all the entries in all the conflicts, as a set."""
    for c in self.conflicts.values():
        for e in c:
            if e:
                yield e.path
```

Reply:
Done